QA/page_objects/fetch_api_response_interview_scheduler.py

```diff
@@ -0,0 +1,43 @@
+"""
+    Fetches the API response from the interview scheduler application
+"""
+import os,sys,time
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from conf import fetch_api_interviewscheduler_conf as conf
+from utils.Option_Parser import Option_Parser
+import requests
+import urllib
+import json
+
+
+
+url=conf.url
+emailid=conf.emailid
+date=conf.date
+
+class fetch_api_response():
+    """
+    Class contains methods to connect to the Interview scheduler website and fetch the free time slots from the api response
+    """
+    def connect_to_website(self):
+        #method to for api call to fetch response from webpage
+        self.config_details={'email':emailid,'date':date}
+        self.api_call=requests.post(url,data=self.config_details)
+        self.free_timeslot_list=self.api_call.json()['free_slots_in_chunks']
```

Comment:
Where are fetching this free slots in chunks?

Reply:
'free slots in chunks' is the name of the list in the api response. So i'm traversing that list and storing the start times and end times into  seperate lists